<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DynamicWidgetClass</class>
 <widget class="QMainWindow" name="DynamicWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1415</width>
    <height>937</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DynamicWidget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="DynamicWidget.qrc">
    <normaloff>:/DynamicWidget/Resource/icon.ico</normaloff>:/DynamicWidget/Resource/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 32, 32);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>100000</width>
     <height>100000</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="MyWindowWidget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 32, 32);
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="MyTitleBar" name="toolBar_TitleBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="MyToolBar" name="toolBar_Tool">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="actionCameraStart">
   <property name="icon">
    <iconset resource="DynamicWidget.qrc">
     <normaloff>:/DynamicWidget/Resource/CameraStart.png</normaloff>:/DynamicWidget/Resource/CameraStart.png</iconset>
   </property>
   <property name="text">
    <string>相机开始工作</string>
   </property>
  </action>
  <action name="actionCameraStop">
   <property name="icon">
    <iconset resource="DynamicWidget.qrc">
     <normaloff>:/DynamicWidget/Resource/CameraStop.png</normaloff>:/DynamicWidget/Resource/CameraStop.png</iconset>
   </property>
   <property name="text">
    <string>相机停止工作</string>
   </property>
  </action>
  <action name="actionAddWindow">
   <property name="icon">
    <iconset resource="DynamicWidget.qrc">
     <normaloff>:/DynamicWidget/Resource/AddWindow.png</normaloff>:/DynamicWidget/Resource/AddWindow.png</iconset>
   </property>
   <property name="text">
    <string>添加窗口</string>
   </property>
  </action>
  <action name="actionSystemSet">
   <property name="icon">
    <iconset resource="DynamicWidget.qrc">
     <normaloff>:/DynamicWidget/Resource/settings.png</normaloff>:/DynamicWidget/Resource/settings.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionFileStart">
   <property name="icon">
    <iconset resource="DynamicWidget.qrc">
     <normaloff>:/DynamicWidget/Resource/FileStart.png</normaloff>:/DynamicWidget/Resource/FileStart.png</iconset>
   </property>
   <property name="text">
    <string>循环读取文件夹</string>
   </property>
  </action>
  <action name="actionFileStop">
   <property name="icon">
    <iconset resource="DynamicWidget.qrc">
     <normaloff>:/DynamicWidget/Resource/FileStop.png</normaloff>:/DynamicWidget/Resource/FileStop.png</iconset>
   </property>
   <property name="text">
    <string>停止读取文件夹</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyTitleBar</class>
   <extends>QToolBar</extends>
   <header>UI/mytitlebar.h</header>
  </customwidget>
  <customwidget>
   <class>MyToolBar</class>
   <extends>QToolBar</extends>
   <header>UI/mytoolbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="DynamicWidget.qrc"/>
 </resources>
 <connections/>
</ui>
